package.json

```diff
@@ -101,6 +97,9 @@
         "typescript": "~5.0.2",
         "yjs": "^13.5.0"
     },
+    "resolutions": {
+        "d3-selection": "1.3.0"
+    },
```

Comment:
Fixes a bug caused by two versions of d3-selection being included in the project, v1.1.1 and v1.3.0. This caused the widget to crash if the volume settings were opened.

Reply:
What was bringing 1.1 and what was bringing 1.3?

Reply:
It looks like `d3-transition` and `d3-zoom` required `d3-selection@npm: ^1.1.0`, both of which are dependencies of `d3`. I'm seeing two types of imports for d3-selection:
```
dependencies:
    d3-selection: 1
```
```
dependencies:
    d3-selection: ^1.1.0
```